jia/jia/auth.py

```
@@ -84,7 +84,9 @@ def google_callback():
           user.email = user_info['email']
           user.picture = user_info['picture']
           user.locale = user_info['locale']
-          user.hd = user_info['hd']
+          # Optional, as per
+          # https://developers.google.com/accounts/docs/OpenIDConnect
+          user.hd = user_info.get('hd', 'notprovided.badtld')
```

that would require a change to the data model + migration, which isn't appropriate for this PR.  do you recommend i delete this change and create an issue for it, or is this temporary fix better than what we have now (silent failures for non-google-hosted accounts)?